interface/HHReweight5D.h

```diff
@@ -3,62 +3,88 @@
 ** author : L. Cadamuro (LLR)
 ** date   : 17 Feb 2017
 ** brief  : Compute a reweight coefficient to transform an input distribution into a general point in kl, kt plane. 31/05/2017: Updated to 5D reweight
+** Updated by D. Zuolo (MIB) and J. Motta (LLR) for Run 2 analisys - Apr 2021
```

Comment:
Maybe add few words saying that the LO reweight has been updated and the NLO reweight as been added, just for future reference.

Reply:
Added a few words and the link to the gitlab page where the procedure is explained in detail